### DISCUSSION TITLE
请问antdv是不是不支持chrome 88一下的浏览器?打包后有css :where 函数

### DISCUSSION THREAD

**Discussion by pork-chop-1** [vueComponent/ant-design-vue#7135] [Q&A] [answered]
chrome 88 一下的浏览器样式无法正常显示，发现它在打包的时候会生成这个：
![image](https://github.com/vueComponent/ant-design-vue/assets/38641727/59fedfad-8001-4a80-b0c3-d67ed0209522)
但是这个:where只支持88以上版本
![image](https://github.com/vueComponent/ant-design-vue/assets/38641727/599486c7-bde2-45f9-838d-3c3beb59ceac)
请问有什么办法可以把这个where编译掉？用vite或者webpack都行

**Comment by shifeng1993** [comment, marked as answer]
可以看文档关于样式处理部分，https://www.antdv.com/docs/vue/compatible-style
